tests/e2e_regression/run.sh

```diff
@@ -69,18 +69,19 @@ testCases=(
 nCases=${#testCases[@]}
 
 # Kill background processes on exit. (In our case the indexer API server.)
+# todo: this doesn't work on MacOS. 
 trap 'trap - SIGTERM && kill -- -$$' SIGINT SIGTERM EXIT
 
 # Start the API server.
 make oasis-indexer
-./oasis-indexer --config="${SCRIPT_DIR}/e2e_config.yaml" serve &
+./oasis-indexer --config="${SCRIPT_DIR}/../api/api-dev.yml" serve &
 while ! curl --silent localhost:8008/v1/ >/dev/null; do
   echo "Waiting for API server to start..."
   sleep 1
 done
 
 # Run the test cases.
-seen=()
+seen=("placeholder") # avoids 'seen[*]: unbound variable' error on zsh
```

Comment:
Just curious: because of the `#!/bin/bash` shebang, don't you have to go to extra trouble to run this in zsh? Alternatively, is there an invocation where you can explicitly invoke the script with bash and avoid zsh compat issues?